posix/mount.c

```diff
@@ -139,7 +139,7 @@ int oe_mount(
     bool locked = false;
     oe_posix_path_t source_path;
     oe_posix_path_t target_path;
-    mount_point_t mount_point;
+    mount_point_t mount_point = {0};
```

Comment:
This should work. But perhaps, initializing all the elements is more readable:
mount_point_t mount_point = {0, 0, 0, 0};

If we were using C99 (I believe we support C98), a better way is to use designated initialization:
mount_point_t mount_point = {.path=0, .path_size=0, .fs=0, flags=0};

Reply:
@anitagov ={0} is a commonly accepted way to 0 out a structure.